### PR TITLE
basic socket connection

### PR DESCRIPTION
**Pull request by agevorgy**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,6 +26,13 @@
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root"></div>
+    <script src="http://localhost:3005/socket.io/socket.io.js"></script>
```
Comment (juanflo) [inline public/index.html]: Why is this here? Maybe better to put it in an action.

Comment (agevorgy) [reply public/index.html]: can I put it in the joinSession component? I have it working there

Comment (juanflo) [reply public/index.html]: Sure
